### PR TITLE
shared: Extract common LogControl CLI code to verb-log-control.h

### PR DESCRIPTION
**Pull request by daandemeyer**
Let's reduce duplication by sharing common logic between all
log-target/log-level verbs.

### CODE REVIEW

File: src/shared/verb-log-control.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#include "alloc-util.h"
+#include "bus-error.h"
+#include "log.h"
+#include "strv.h"
+#include "syslog-util.h"
+#include "verb-log-control.h"
+
+int verb_log_control_common(sd_bus *bus, const char *destination, const char *verb, const char *value) {
+        assert(bus);
+        assert(endswith(verb, "log-level") || endswith(verb, "log-target"));
```
Comment (bluca) [inline src/shared/verb-log-control.c]: why the asserts before the declarations?

**Review by bluca — APPROVED** [review]
lgtm, one nit

File: src/shared/verb-log-control.c (hunk 1 above)
Comment (daandemeyer) [reply src/shared/verb-log-control.c]: This function is more or less copy pasted from `log_setting_internal` which did the same so I kept it. Will change to the usual style.

**Comment by yuwata** [comment]
LGTM too.

**Pull request merged**